lib/BoundaryValueDiffEqCore/src/BoundaryValueDiffEqCore.jl

```diff
@@ -18,9 +18,10 @@ import Logging
 using NonlinearSolveFirstOrder: NonlinearSolvePolyAlgorithm
 import LineSearch: BackTracking
 import RecursiveArrayTools: VectorOfArray, DiffEqArray
-import SciMLBase: AbstractDiffEqInterpolation, StandardBVProblem, __solve, _unwrap_val
+import SciMLBase: AbstractBVProblem, AbstractDiffEqInterpolation, StandardBVProblem,
+                  __solve, _unwrap_val
 
-@reexport using ADTypes, NonlinearSolveFirstOrder, SparseDiffTools, SciMLBase
+@reexport using NonlinearSolveFirstOrder, SparseDiffTools, SciMLBase
```

Comment:
do you still need SparseDiffTools?

Reply:
The Shooting methods is still working on progress, which require SparseDiffTools.jl